frontend/package.json

```diff
@@ -33,7 +33,10 @@
     "test:fix": "eslint --ext .json,.js,.ts,.jsx,.tsx ./src --fix",
     "test:lint": "eslint --max-warnings 0 --ext .json,.js,.ts,.jsx,.tsx ./src",
     "test:local": "jest --watchAll",
-    "test:type-check": "tsc --noEmit"
+    "test:type-check": "tsc --noEmit",
+    "test:storybook": "test-storybook --verbose",
+    "storybook": "start-storybook -p 6006 -s ./config",
+    "build-storybook": "build-storybook"
```

Comment:
What would be the use-case for building it?

Reply:
we would build it if we wanted to deploy a hosted version of storybook. (could be a form of docs)